MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -50,12 +53,25 @@ typedef struct {
   ///
   /// The total STM_HEADER should be 4K.
   ///
+  
+  // Pad to take up 1KB of space
+  UINT8     Reserved2[SIZE_1KB - 5 * sizeof (UINT32) - 3 * sizeof (UINT8) - sizeof (UINT16) - sizeof (STM_FEAT)];
 } SOFTWARE_STM_HEADER;
 
+// MU_CHANGE - [START]: Add new CPU information header for STM
 typedef struct {
-  MSEG_HEADER            HwStmHdr;
-  SOFTWARE_STM_HEADER    SwStmHdr;
+  UINT32    Signature;
+  UINT32    NumberOfCpus;
+  /// Pad to take up 1KB of space
+  UINT8     Reserved[SIZE_2KB - 2 * sizeof (UINT32)];

```

Comment:
Shouldn't this be 1KB - 2 UINT32?

Reply:
That's what it currently is.  Also it moved to a define above.